server/test/christmas-tree-controllers.es6

```diff
@@ -14,6 +15,7 @@ let permitId;
 let invalidPermitId = 'xxxxx';
 let paygovToken;
 let tcsAppID;
+let today = moment(new Date()).format('YYYY-MM-DD');
```

Comment:
the dates within the db are stored as UTC, correct?

Reply:
this will change once we pass a js timestamp as noted above

Reply:
correct @lauraGgit, db dates are stored in UTC. This test is to get the report on that day.

Reply:
cool cool, just checking!